.github/config.yml

```
@@ -0,0 +1,29 @@
+  Be sure to review our contributing guidelines to ensure a smooth merge process.  
+  ![High Five](https://media.giphy.com/media/3oEjI6SIIHBdRxXI40/giphy.gif)
+
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted on pull requests merged by a first-time user
+firstPRMergeComment: >
+  🥳 **Congratulations, @{{user.login}}!** 🥳  
+  Your first pull request has been merged! Thank you for your contribution—we can't wait to see what you'll do next.  
+  ![Celebration](https://media.giphy.com/media/l4FGuhL4U2WyjdkaY/giphy.gif)
+
+# It is recommended to include as many GIFs and emojis as possible
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)